README.md

```diff
@@ -1,11 +1,11 @@
 # SuperTuxKart
 [![Build Status](https://travis-ci.org/supertuxkart/stk-code.svg?branch=master)](https://travis-ci.org/supertuxkart/stk-code)
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/supertuxkart/stk-code?svg=true&branch=master)](https://ci.appveyor.com/project/supertuxkart/stk-code)
-[![#supertuxkart on the freenode IRC network](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://webchat.freenode.net/?channels=supertuxkart)
+[![#supertuxkart on the freenode IRC network](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://kiwiirc.com/client/irc.freenode.net/#supertuxkart)
```

Comment:
why would you change that? we used the official freenode client

Reply:
It's very hard to keep track of what's being said vs connect/disconnect messages, making it very easy to miss replies, etc.
![](http://i.imgur.com/Gwpnz6D.png)
vs
![](http://i.imgur.com/Ucfol46.png)
(Plus having to do a captcha on qwebirc is annoying, KiwiIRC doesn't have a captcha ;)